mesh/mesh.hpp

```diff
@@ -669,19 +721,15 @@ class Mesh
       Finalize(true); // refine = true
    }
 
-   /** Creates mesh for the rectangle [0,sx]x[0,sy], divided into nx*ny
-       quadrilaterals if type = QUADRILATERAL or into 2*nx*ny triangles if
-       type = TRIANGLE. If generate_edges = 0 (default) edges are not generated,
-       if 1 edges are generated. If scf_ordering = true (default), elements are
-       ordered along a space-filling curve, instead of row by row. */
+   /// See @a MakeCartesian2D.
    Mesh(int nx, int ny, Element::Type type, bool generate_edges = false,
         double sx = 1.0, double sy = 1.0, bool sfc_ordering = true)
    {
       Make2D(nx, ny, type, sx, sy, generate_edges, sfc_ordering);
       Finalize(true); // refine = true
    }
 
-   /** Creates 1D mesh , divided into n equal intervals. */
+   /// See @a MakeCartesian1D.
```

Comment:
Maybe change comment to "Deprecated: see @a MakeCartesian1D"?

Reply:
Done.